mbedtls/tests/hyper.rs

```diff
@@ -283,9 +280,6 @@ mod tests {
         };
         config.set_dbg_callback(dbg_callback);
 
-        #[cfg(not(target_env = "sgx"))]
-        unsafe { set_global_debug_threshold(1); }
```

Comment:
please keep this if possible as it's been very useful debugging code when unit test fails.

Reply:
That would require limiting this test to run only when a certain feature is set, limiting its usefulness.